xds/src/main/java/io/grpc/xds/XdsServerBuilder.java

```diff
@@ -56,9 +56,10 @@ private XdsServerBuilder(NettyServerBuilder nettyDelegate, int port) {
     return delegate;
   }
 
-  /** Set the {@link ErrorNotifier}. Pass null to unset a previously set value. */
-  public XdsServerBuilder errorNotifier(ErrorNotifier errorNotifier) {
-    this.errorNotifier = errorNotifier;
+  /** Set the {@link XdsServingStatusListener}. Pass null to unset a previously set value. */
```

Comment:
What's the usecase to unset a previously value? I see `builderServer()` will throw NPE if `xdsServingStatusListener ` is not set.

Reply:
Good catch. With the `DefaultListener` there is no use case for this. Fixing